packages/govern/scripts/start.e2e.sh

```diff
@@ -4,6 +4,9 @@
 cd ../govern-server/
 nohup yarn dev >>/dev/null 2>>/dev/null &
 
+# Check if the govern-server is running
+while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' localhost:3000/.well-known/apollo/server-health)" != "200" ]]; do sleep 5; done
```

Comment:
Let's avoid complicated bash scripts whenever we can, as soon as we have `if` / `while` I prefer javascript:
```suggestion
npx wait-on localhost:3000/.well-known/apollo/server-health --timeout 20000
```
Also can you add this into top `package.json` with something like `test:server`